.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+jobs = auto 
+max-line-length = 120 
+select=E901,E999,F821,F822,F823,I100,I101,I201,I202
+ignore=E203,W503,C0330
```

Comment:
For discussion, these are the included/excluded errors:

```
Included:
SyntaxError: invalid syntax (E901)
SyntaxError: invalid syntax (E999)
Undefined name name (F821)
Undefined name name in __all__ (F822)
Local variable name ... referenced before assignment (F823)
Your import statements are in the wrong order (I100)
The names in your from import are in the wrong order (I101)
Missing newline between import groups (I201)
Additional newline in a group of imports (I202)

Excluded:
Whitespace before ':' (E203)
Line break occurred before a binary operator (W503)
Wrong hanging indentation before block (C0330)
```

Reply:
For comparison, in IPv8 we only exclude `E501,E722,E731,E741,F403,F405,W503`:

```
Excluded:
Line too long (E501)
Do not use bare except (E722)
Do not assign a lambda expression, use a def (E731)
Do not use variables named 'l', 'o', or 'i' (E741)
'from module import *' used; unable to detect undefined names (F403)
Name may be undefined, or defined from star imports: module (F405)
Line break occurred before a binary operator (W503)
```

Reply:
The list of checks I used for this PR is copied from our Jenkins Flake8 tester, BTW.